autoload/clap/provider/quickfix.vim

```diff
@@ -0,0 +1,35 @@
+" Author: KITAGAWA Yasutaka <kit494way@gmail.com>
+" Description: List the entries of the quickfix list.
+
+let s:save_cpo = &cpoptions
+set cpoptions&vim
+
+let s:quickfix = {}
+
+function! s:quickfix.source() abort
+  " Ignore quickfix list entries with non-existing buffer number.
+  let qflist = filter(getqflist(), 'v:val["bufnr"]')
+
+  return map(getqflist(), 's:to_grepformat(v:val)')
+endfunction
+
+function! s:to_grepformat(quickfix) abort
+  let path = bufname(a:quickfix['bufnr'])
+  let line_col = a:quickfix['lnum'].' col '.a:quickfix['col']
+  return path.'|'.line_col.'|'.a:quickfix['text']
+endfunction
+
+function! s:quickfix.sink(selected) abort
+  let [fpath, line_col] = split(a:selected, '|')[:1]
+  let [lnum, column] = split(line_col, ' col ')
+
+  execute 'edit' fpath
+  noautocmd call cursor(lnum, column)
+endfunction
+
+let s:quickfix.enable_rooter = v:true
+
```

Comment:
Try adding `let s:quickfix.on_enter = { -> g:clap.display.setbufvar('&ft', 'qf')}` for getting the qf highlight.

Reply:
Thanks for the suggestion. I'll try this.